frontend/packages/rhoas-plugin/api-mock/README.md

```diff
@@ -0,0 +1,7 @@
+## Mocking capability for RHOAS Operator
```

Comment:
This will be removed at some point - We have that for demo purposes and to simulate errors.